pkgs/servers/sql/mariadb/default.nix

```diff
@@ -224,7 +216,7 @@ galera = stdenv.mkDerivation rec {
   version = "25.3.23";
 
   src = fetchurl {
-    url = "https://mirrors.nxthost.com/mariadb/mariadb-10.2.14/galera-${version}/src/galera-${version}.tar.gz";
+    url = "https://mirrors.nxthost.com/mariadb/mariadb-10.3.8/galera-${version}/src/galera-${version}.tar.gz";
```

Comment:
Does the old url not work anymore?

Reply:
Working. Just changed the link to the current version MariaDB

Reply:
So it's a version update? If so you also need to update the hash, and mark it as such in the commit message

Reply:
No, these are identical versions.